gulpfile.js

```diff
@@ -106,6 +118,25 @@ function errorHandler(error) {
 }
 
 /**
+ * Task: `build-pl`
+ * Builds the bundled pattern library (if necessary)
+ */
+gulp.task('build-pl', function (done) {
+    if (buildPatternLibrary) {
+        exec('{ cd ' + patternLibrary + ' && npm install ;} 2>&1', function (error, stdout) {
```

Comment:
Would it make sense to simply include the custom PL via a submodule? or do you thinking doing a separate npm install is useful?